packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -200,6 +200,7 @@ function AppLayoutSidebarNavListItem({
 						{item.items?.map?.((item) => (
 							<AppLayoutSidebarNavListItem
 								activePath={activePath}
+								background={background}
```

Comment:
Really annoying how when you technically get the type wrong, TS can't help you. We should have noticed that `background` wasn't being passed through, but since it was optional, it was easily missed… probably because we trust TS too much.

Reply:
Over the years, I've definitely fallen into the camp of making types the source of truth. Infer as much as possible but trust the contract.